### DISCUSSION TITLE
New Cypress Allure 2.14.1 Is not Supporting Import Style

### DISCUSSION THREAD

**Discussion by raajrsaravanan** [allure-framework/allure2#2408] [Questions & Support] [answered]
Import allureCypress from 'allure-cypress/reporter' is not supporting due to missing export info in Package.json. Could anyone please fix this to use Import Style in ES Project

<img width="1106" alt="Screenshot 2024-03-09 at 17 37 10" src="https://github.com/allure-framework/allure2/assets/115946632/aae863c1-fa1b-46b1-86a2-cc286a8992ed">
<img width="293" alt="Screenshot 2024-03-09 at 17 37 34" src="https://github.com/allure-framework/allure2/assets/115946632/901b6db0-7d9f-487e-a126-30d4111d36e0">
<img width="421" alt="Screenshot 2024-03-09 at 17 38 28" src="https://github.com/allure-framework/allure2/assets/115946632/e2cfb587-6938-4463-aa3b-019de319d35c">

**Comment by raajrsaravanan** [comment]
@baev @epszaw @eroshenkoam @sseliverstov @vovsemenv If any one of you take this changes asap. would be great for some of people who is using ES module projects. Thanks a lot

**Reply by baev** [reply]
We're working on this request. However, it requires significant changes to the `allure-js-commons`. The WIP PR: https://github.com/allure-framework/allure-js/pull/905

**Reply by epszaw** [reply]
The PR is ready. Just wait until we're done reviewing the code and release a new version 🙏

**Reply by epszaw** [reply, marked as answer]
So, the PR was merged and the last version (`2.15.0`) for `allure-cypress` should work both with `esm` and `cjs` modules.
